src/pfs/pfs.proto

```diff
@@ -21,6 +21,7 @@ message Repo {
   option (gogoproto.goproto_stringer) = false;
   string name = 1;
   string type = 2;
+  Project project = 3;
```

Comment:
Should this be part of this PR, or one implementing CORE-173?

Reply:
i think it could be in either. I threw it in here to try to list out the general proto structure / relationships